File: 0088 merge-sorted-array/solution.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        int len = m + n - 1;
        m--;
        n--;
        while (n >= 0) {
            if (m >= 0 && nums1[m] > nums2[n])
                nums1[len--] = nums1[m--];
            else
                nums1[len--] = nums2[n--];
        }
    }
}

File: 0088 merge-sorted-array/solution.cs
public class Solution {
    public void Merge(int[] nums1, int m, int[] nums2, int n) {
        int l=m+n-1;
        m--;
        n--;
        while(n>=0) {
            if(m>=0 && nums1[m]>nums2[n]) nums1[l--]=nums1[m--];
            else nums1[l--]=nums2[n--];
        }
    }
}
